lib/stat_tracker.rb

```diff
@@ -315,9 +315,57 @@ def worst_coach(season)
     coach_losses.max_by { |coach| losses_by_coach[coach] }
```

Comment:
@sciencefixion this is the only part of `worst_coach` that doesn't make sense to me so far... I think you were trying to say 
```
loss_count = losses_by_coach.max_by { |coach, count| count }
```
and then you can return the coach in question by calling 
`losses_by_coach.key(loss_count)`

Reply:
`hash.key(value)` returns the `key` that corresponds to given `value`